Tests/Controller/Annotations/AbstractParamTest.php

```diff
@@ -28,7 +28,6 @@ public function setUp()
     public function testInterface()
     {
         $this->assertInstanceOf('FOS\RestBundle\Controller\Annotations\ParamInterface', $this->param);
-        $this->assertInstanceOf('Symfony\Component\DependencyInjection\ContainerAware', $this->param);
```

Comment:
is there an elegant way to check for the trait? I guess one could check for the method .. then again what was the value of this assertion to begin with?